Template/Tag/CustomHtmlTag.php

```diff
@@ -33,23 +34,23 @@ public function getParameters()
     {
         return array(
             $this->makeSetting('customHtml', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
-                $field->title = 'Custom HTML';
+                $field->title = Piwik::translate('TagManager_CustomHtmlTagTitle');
```

Comment:
It looks like `TagManager_CustomHtmlTagName` already exists and has the same value.